### PR TITLE
Port examples from undocumented kernel

### PR DESCRIPTION
**Pull request by jonpsy**
Hey! So I've added like 7 examples from undocumented python codes. I compiled them all and checked in cpp. They all work fine. 

Reviews? @vigsterkr  @gf712

### CODE REVIEW

File: examples/meta/src/kernel/gaussian.sg.in [hunk 1]
```diff
@@ -0,0 +1,18 @@
+File traindat = csv_file("@SHOGUN_DATA@/fm_train_real.dat")
+File testdat = csv_file("@SHOGUN_DATA@/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Features feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel gaussian_kernel = kernel("GaussianKernel", log_width=2.00)
```
Comment (gf712) [inline examples/meta/src/kernel/gaussian.sg.in]: minor, but this could just be `2.0` no?

File: examples/meta/src/kernel/const.sg.in [hunk 2]
```diff
@@ -0,0 +1,18 @@
+File traindat = csv_file("@SHOGUN_DATA@/fm_train_real.dat")
+File testdat = csv_file("@SHOGUN_DATA@/fm_test_real.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+Features feats_test = features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel const_kernel = kernel("ConstKernel", const_value=23.00)
```
Comment (gf712) [inline examples/meta/src/kernel/const.sg.in]: same here, could this not just be `23.0`?

File: examples/meta/src/kernel/gaussian.sg.in (hunk 1 above)
Comment (jonpsy) [reply examples/meta/src/kernel/gaussian.sg.in]: It asked for double, so i added for more precision. 

But it's your call 🙂

File: examples/meta/src/kernel/const.sg.in (hunk 2 above)
Comment (jonpsy) [reply examples/meta/src/kernel/const.sg.in]: Done

File: examples/meta/src/kernel/gaussian.sg.in (hunk 1 above)
Comment (jonpsy) [reply examples/meta/src/kernel/gaussian.sg.in]: Done

Comment (gf712) [reply examples/meta/src/kernel/gaussian.sg.in]: hmm that's not what double precision means! Double precision is usually a 64-bit representation of a floating point type, where as single will be 32-bit. This is unrelated to the number of 0s you put here :)

Comment (jonpsy) [reply examples/meta/src/kernel/gaussian.sg.in]: You're right I guess I need to revisit basic data types xD 😁

Comment (jonpsy) [reply examples/meta/src/kernel/gaussian.sg.in]: Shall I git squash?   With better commit message ?

Comment (gf712) [reply examples/meta/src/kernel/gaussian.sg.in]: No that's fine, can do that when it's merged

Comment (jonpsy) [reply examples/meta/src/kernel/gaussian.sg.in]: Thanks for that comment. I read up on double and revised the concepts again.

**Comment by gf712** [comment]
@NanuSai could you show what you have in `/build/examples/meta/java/kernel/const.java`? Seems like it's not passing in the CI

**Comment by jonpsy** [comment]
https://pastebin.com/UUbX5dE7

**Comment by jonpsy** [comment]
> https://pastebin.com/UUbX5dE7

Ofcourse ! It is using ``` public class const ```, we can't have const as a class name ! . I shall be changing ```const.sg.in``` to ```consant.sg.in```.

**Comment by vigsterkr** [comment]
@NanuSai cool! now comes the part to update the shogun-data repository with some integration data, see here: https://github.com/shogun-toolbox/shogun/blob/develop/doc/readme/DEVELOPING.md#adding-tests-2

**Comment by jonpsy** [comment]
@vigsterkr Done, do check out my PR on shogun-data repo

**Comment by vigsterkr** [comment]
@NanuSai ok so now you'll need to update this pr that uses that commit from the data :)

**Comment by vigsterkr** [comment]
ok great lets see now CI

**Comment by jonpsy** [comment]
@vigsterkr Looks like it's done 💪

**Comment by vigsterkr** [comment]
@NanuSai mmm seems the last change in data unfortunately makes this unmergable... could you rebase and fix the conflicts plz?

**Comment by jonpsy** [comment]
Hi @vigsterkr , It seems like the error originates after the commit ``` Added .dat files ```. There has been some changes in the ```data ``` file as "both modified". I'll read up on them and resolve conflicts

**Comment by vigsterkr** [comment]
rebase the shogun-data PR and resolve the conflicts if there are any and then rebase this branch and resolve the conflicts...

**Comment by jonpsy** [comment]
The conflicts seems to be resolved now. Let's wait for CI :)

**Comment by gf712** [comment]
@NanuSai did you rebase the data submodule? It seems like the feature_importance data is missing?

**Comment by jonpsy** [comment]
@gf712, Yes just did that thanks for pointing it out. Hopefully nothing goes wrong now, haha!

**Pull request merged**